.github/workflows/base-branch-check.yml

```
@@ -0,0 +1,18 @@
+name: Pull Requests
+
+on:
+  pull_request_target: # By default, a workflow only runs when a pull_request_target's activity type is opened, synchronize, or reopened.
+    branches:
+      - master # Run only on master branch
+
+jobs:
+  base_branch_check:
+    runs-on: ubuntu-latest
+    name: Base Branch Check
+    steps:
+      - name: Comment PR
```

```suggestion
      - name: Get branch names
        id: branch-name
        uses: tj-actions/branch-names@v5.1
      - name: Comment PR
        if: steps.branch-name.outputs.head_ref_branch != 'development'
```

Sorry to throw a spanner in the works here, is there any value in adding this [action](https://github.com/tj-actions/branch-names) to the CI run here - which would prevent this comment from appearing when we merge `dev -> master` for release (I think)

---

I guess also we don't want it if branch starts with `release/` (but that should also be trivial to add as a conditional on the `if:` bit